docker/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
             - "50051:50051"
             - "14540:14540/udp"
     gazebo_sitl_headless:
-        image: jonasvautherin/px4-gazebo-headless:v1.9.2
+        image: jonasvautherin/px4-gazebo-headless:v1.10.2
```

Comment:
Does that work? Latest version is `1.10.1`

Reply:
There is a tag though: https://github.com/PX4/Firmware/releases/tag/v1.10.2
Strange...

Reply:
Yes, but this has to be done manually in this repo :-). We just haven't done it, so the latest `px4-gazebo-headless` tag is `1.10.1` :see_no_evil:

Reply:
I didn’t test it, sorry

Reply:
Oh! Let me change it then.

Reply:
https://github.com/mavlink/MAVSDK-JavaScript/pull/13